src/domUpdates.js

```diff
@@ -239,6 +239,16 @@ function createTable() {
       <td class="button-column hidden-row"></td>
     </tr>`
   }
+  const minusButtons = document.querySelectorAll('.minus-js');
```

Comment:
Just about to circle back to this. As is, it didn't work. This is our next issue to tackle.